src/pool/srv_layout.h

```diff
@@ -67,6 +67,7 @@ extern d_iov_t ds_pool_attr_user;		/* pool user attributes KVS */
 struct pool_hdl {
 	uint64_t	ph_flags;
 	uint64_t	ph_sec_capas;
+	char		ph_machine[MAXHOSTNAMELEN+1];
```

Comment:
I will bump these once we branch for 2.0/2.2

Reply:
The concern with this PR when it lands to master is that it will effectively lower / revert DAOS_POOL_MD_VERSION=4 and DAOS_POOL_MD_VERSION_LOW=4 to 3 and 2.

So how will this work when a release 2.0 branch is created. Will that 2.0 branch have versions=4? And then this PR will land to master and effectively revert to versions=3/2?

Reply:
When we branch 2.0 and master this will have to have master merged into it. I don't intend on spending time on it until that happens since we have lots of other bugs and issues that need to be resolved for 2.0.